app/src/main/java/com/github/orkest/Model/FireStoreDatabaseAPI.kt

```diff
@@ -203,6 +219,28 @@ class FireStoreDatabaseAPI {
         return future
     }
 
+
+    /**
+     * @param us
+     */
+    fun getPostCommentsFromDataBase(post_username: String, post_date: String): CompletableFuture<List<Comment>>{
```

Comment:
In future sprints, we should see if there is a way to fctorize your deserializing method (comments) and the ones deserializing posts and profiles as they do the same except for the path and the document type to be deserialized